js/midtrans.admin.js

```diff
@@ -0,0 +1,153 @@
+(function ($, Drupal, drupalSettings) {
```

Comment:
Code on this file is looks longer and more complex than what it actually do which I suppose is: `Dynamically put sandbox/production text label on API keys field, according to selected API env mode. For each of Midtrans Modules (fullpayment, installment, promo etc)`.

To make it easier to understand either:
- Put sentence above as comments in this file, so reader can quickly understand the whole code in just 1 sentence
- Shorten the code by using common/shared function. I see redundant code snippet/blocks on the implementation (can be extracted into functions), which the difference is the HTML class names (us it as params to the function)